webapp/src/css/inbox.less

```diff
@@ -627,7 +632,7 @@ body {
 }
 
 .loading-status {
-  margin: 10px 10px @bottom-navbar-height;
+  margin: 10px 10px calc(@bottom-navbar-height + 10px);
```

Comment:
We had to add a bottom margin in some pages, are they all the same 10 to 15px? Which can be a good variable
`@navbar-page-margin`, and extract these to 1 rule. For example:

```
.page-1 .loading-status
.page-2 .list-component
.page-X ..... { <---- next time we need to add this margin, we just add the page and the element to attach the margin
  margin-bottom: calc(@bottom-navbar-height + @navbar-page-margin);
}
```